src/lasp_transmission_instrumentation.erl

```diff
@@ -40,9 +40,9 @@
 -include("lasp.hrl").
 
 %% State record.
--record(state, {type,
+-record(state, {entity,
                 tref,
-                size=0,
+                size_per_type=orddict:new(),
```

Comment:
That sounds good to me.

How will you distinguish in the output log file that you generate at the end of the execution?